app/grandchallenge/hanging_protocols/views.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.messages.views import SuccessMessageMixin
+from django.views.generic import CreateView, DetailView, ListView, UpdateView
+from guardian.mixins import LoginRequiredMixin
+from guardian.mixins import (
+    PermissionRequiredMixin as ObjectPermissionRequiredMixin,
+)
+
+from grandchallenge.hanging_protocols.forms import HangingProtocolForm
+from grandchallenge.hanging_protocols.models import HangingProtocol
+from grandchallenge.subdomains.utils import reverse
+
+
+class HangingProtocolList(LoginRequiredMixin, ListView):
+    model = HangingProtocol
+    paginate_by = 50
+
+
+class HangingProtocolCreate(
+    LoginRequiredMixin, SuccessMessageMixin, CreateView

```

Comment:
I think we should limit who can create protocols like we do the workstation configs (etc), can use the global `create_hangingprotocol` permission.